thirdeye/thirdeye-frontend/app/shared/dimensionAnalysisTableConfig.js

```diff
@@ -1,13 +1,15 @@
+const COLUMN_CLASS = 'rootcause-dimensions-table__column';
+
 /**
  * Dynamic grouped headers for RCA advanced dimension analysis table.
  * Colspan for 'dimensions' group dependent on user-selected analysis 'depth' level.
  */
 export const groupedHeaders = (advDimensionCount, overallChange) => [
   [
-    {title: 'Dimensions', class: 'advanced-dimensions-table__header', colspan: advDimensionCount},
-    {title: '', class: 'advanced-dimensions-table__header', colspan: 1},
-    {title: `Overall Change ${overallChange}`, class: 'advanced-dimensions-table__header', colspan: 1},
-    {title: '', class: 'advanced-dimensions-table__header', colspan: 2}
+    {title: 'Dimensions', className: 'rootcause-dimensions-table__header', colspan: advDimensionCount},
```

Comment:
I think as this is a `shared` component, the class prefix `advanced-dimensions` was more appropriate than `rootcause-dimensions`. Unless you plan to not have it be reuse for other pages that aren't `rootcause`?

Reply:
Will consider.